roshan-bajgain/math-magicians
# Testing

## Math Magicians: tests
- Set up React Testing Library
- Create unit tests for all React components
- Write unit tests for the files operate.js and calculate.js using Jest.
- Run your full tests suite using Jest. All tests should pass.
